pkgs/development/python-modules/jinja2-strcase/default.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  freezegun,
+  jinja2,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "jinja2-strcase";
+  version = "0.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-2Qw3971A00WqzI94sIf2bmxapMloqyOnkVc/z3VsM3k=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
```

```suggestion
  build-system = [ setuptools ];
```
Please use `build-system` instead.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters